homeassistant/components/modbus/climate.py

```diff
@@ -146,7 +146,6 @@ def should_poll(self):
 
         False if entity pushes its state to HA.
         """
-
```

Comment:
This has nothing to do with changing constants (which I see as a good idea), the empty line is design,  please remove this change.